### PR TITLE
kb(grid):added kb for changing built-in icons in the hierarchical grid

### PR DESCRIPTION
**Pull request by xristianstefanov**

### CODE REVIEW

File: knowledge-base/grid-change-expand-collapse-icons.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Description
+How to change the expand/collapse icons in a Hierarchical Grid?
+I want to change the built-in plus/minus icons in a Hierarchical Grid. 
+
+## Solution
+You can change the expand/collapse [icons]({%slug general-information/font-icons%}) in the Hierarchical Grid by overriding the built-in plus/minus icons with other icons using custom CSS rules. In addition, you can use the Class parameter of the Grid to add custom CSS Class and modify a specific instance of the Grid, instead of all instances on the page. The code block below demonstrates how to achieve the described approach.
+
+````CSHTML
+@*Change the icons in the Hierarchical Grid*@
+<style>
```
Comment (marin-bratanov) [inline knowledge-base/grid-change-expand-collapse-icons.md:33]: Add a blank line between the comment and the code, it makes it more readable

File: knowledge-base/grid-change-expand-collapse-icons.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	</tbody>
+</table>
+
+
+## Description
+How to change the expand/collapse icons in a Hierarchical Grid?
+I want to change the built-in plus/minus icons in a Hierarchical Grid. 
+
+## Solution
+You can change the expand/collapse [icons]({%slug general-information/font-icons%}) in the Hierarchical Grid by overriding the built-in plus/minus icons with other icons using custom CSS rules. In addition, you can use the Class parameter of the Grid to add custom CSS Class and modify a specific instance of the Grid, instead of all instances on the page. The code block below demonstrates how to achieve the described approach.
```
Comment (marin-bratanov) [inline knowledge-base/grid-change-expand-collapse-icons.md:29]: Consider adding a Notes section at the end to tell people how to get those icons (inspect a rendered icon they want to use and get its content), and to also mention they can use custom icons that are not from the Telerik font (e.g., icons from bootstrap, openiconic and so on) by also specifying the desired font name in addition to the content.

**Review by marin-bratanov — APPROVED** [review]

File: knowledge-base/grid-change-expand-collapse-icons.md (hunk 1 above)
Comment (xristianstefanov) [reply knowledge-base/grid-change-expand-collapse-icons.md:34]: Done

File: knowledge-base/grid-change-expand-collapse-icons.md (hunk 2 above)
Comment (xristianstefanov) [reply knowledge-base/grid-change-expand-collapse-icons.md:29]: Done

**Pull request merged**